src/test/java/net/revelc/code/impsort/ImpSortTest.java

```diff
@@ -30,7 +34,27 @@
   @Test
   public void testSort() throws IOException {
     Path p = Paths.get(System.getProperty("user.dir"), "src", "it", "plugin-test", "src", "test", "java", "net", "revelc", "code", "imp", "PluginIT.java");
-    new ImpSort(eclipseDefaults).parseFile(p);
+    new ImpSort(eclipseDefaults, false).parseFile(p);
```

Comment:
I'm wondering if the functionality should be added to the Grouper instead of directly in the ImpSort class, since the Grouper tends to handle most of the other configuration options. It would save from adding a boolean flag as a parameter, which I think tends to be not very elegant. (See https://stackoverflow.com/questions/6107221/alternatives-to-passing-a-flag-into-a-method for some other alternatives to the boolean, if Grouper is not a good fit.)

Reply:
Some observation on this:
1. The `extractTokens` function that walks through the `CompilationUnit` and extracts the tokens, should be in the `ImpSort` class. The alternative would be to pass the `CompilationUnit` to the Grouper, and let the Grouper extract the tokens, but this feels wrong.
2. Since we only want to run `extractTokens` if `removeUnused` is `true`, the `removeUnused` flag must be available for the `ImpSort` class to check.

I think I'll take some time to consider the alternatives to the boolean, as I think it is hard to move this functionality out of ImpSort in a straight-forward way.

Reply:
Changes to this could be deferred to a future design change. I wouldn't hold up merging on the this comment. Just thought it was worth asking. :smiley_cat: